plugins/Vestige/Vestige.cpp

```diff
@@ -961,11 +970,16 @@ ManageVestigeInstrumentView::ManageVestigeInstrumentView( Instrument * _instrume
 	l->addWidget( m_displayAutomatedOnly, 0, 1, 1, 2, Qt::AlignLeft );
 
 
-	m_closeButton = new QPushButton( tr( "    Close    " ), widget );
```

Comment:
The Close button didn't really have a reason to be there since you could just click the window's close button to do the same thing. So I used the space it occupied for the pin connector button.